modules/rosa-accessing-your-cluster.adoc

```diff
@@ -106,13 +106,14 @@ $ rosa describe cluster --cluster=<cluster_name>
 Name:        rh-rosa-test-cluster1
 ID:          1de87g7c30g75qechgh7l5b2bha6r04e
 External ID: 34322be7-b2a7-45c2-af39-2c684ce624e1
-API URL:     https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443
+API URL:     https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443 #<1>
 Console URL: https://console-openshift-console.apps.rh-rosa-test-cluster1.j9n4.s1.devshift.org
 Nodes:       Master: 3, Infra: 3, Compute: 4
 Region:      us-east-2
 State:       ready
 Created:     May 27, 2020
 ----
+<1> For {hcp-title-first} cluster, the port number should be `443`.
```

Comment:
Should this read as "For {hcp-title-first} clusters" or "For a {hcp-title-first} cluster"?

Might also consider rephrasing for clarity. Something like: "For {hcp-title-first} clusters, use port 443."

Reply:
Please see https://github.com/openshift/openshift-docs/pull/72868#discussion_r1524263756.